packages/core/src/dialects/mysql/query-generator-typescript.ts

```diff
@@ -53,6 +54,29 @@ export class MySqlQueryGeneratorTypeScript extends AbstractQueryGenerator {
     return `DROP INDEX ${this.quoteIdentifier(indexName)} ON ${this.quoteTable(tableName)}`;
   }
 
+  getForeignKeyQuery(tableName: TableNameOrModel, columnName?: string) {
+    const table = this.extractTableDetails(tableName);
+
+    return joinSQLFragments([
+      'SELECT CONSTRAINT_NAME as constraintName,',
+      'CONSTRAINT_SCHEMA as constraintSchema,',
+      'CONSTRAINT_CATALOG as constraintCatalog,',
```

Comment:
Should we load these? According to the documentation it's always going to be `'def'` because there are no databases in mysql